internal/querycoord/task_scheduler.go

```diff
@@ -463,6 +463,8 @@ func (scheduler *TaskScheduler) processTask(t task) error {
 				protoSize = proto.Size(childTask.(*loadSegmentTask).LoadSegmentsRequest)
 			case commonpb.MsgType_WatchDmChannels:
 				protoSize = proto.Size(childTask.(*watchDmChannelTask).WatchDmChannelsRequest)
+			case commonpb.MsgType_WatchDeltaChannels:
+				protoSize = proto.Size(childTask.(*watchDeltaChannelTask).WatchDeltaChannelsRequest)
```

Comment:
reloadFromKV and wait waitActivateTaskDone function should deal WatchDeltaChannelsRequest